examples/player/simid_player.js

```diff
@@ -131,6 +131,30 @@ class SimidPlayer {
     });
   }
 
+  /**
+   * Player closes the ad and SIMID creative.
+   */
+  playerCloseAd() {
+    this.stopAd();
+    this.sendLog("Ad stopped by player");
+  }
+
+  /**
+   * Player skips the ad.
+   */
+  playerSkipAd() {
```

Comment:
If we can get rid of these functions that would be more readable I think. Did you have them here for debugging purposes?

Reply:
got rid of them